test/index.test.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const Bluebird = require( '../lambda-src/node_modules/bluebird')
+var expect = require( '../lambda-src/node_modules/chai' ).expect;
```

Comment:
const?

Reply:
also, inconsistent use of trailing semicolons

Reply:
not using semicolons anymore.